File: connectors/connector-trello/src/main/scala/com/taskadapter/connector/trello/TrelloApiFactory.scala
package com.taskadapter.connector.trello

import com.julienvey.trello.Trello
import com.julienvey.trello.impl.TrelloImpl
import com.julienvey.trello.impl.http.ApacheHttpClient

object TrelloApiFactory {
  def createApi(appKey: String, token: String): Trello = {
    val a = new ApacheHttpClient
    new TrelloImpl(appKey, token, a)
  }
}

File: connectors/connector-trello/src/main/java/com/taskadapter/connector/trello/TrelloApiFactory.java
package com.taskadapter.connector.trello;

import com.julienvey.trello.Trello;
import com.julienvey.trello.impl.TrelloImpl;
import com.julienvey.trello.impl.http.ApacheHttpClient;

public class TrelloApiFactory {
    static Trello createApi(String appKey, String token) {
        var a = new ApacheHttpClient();
        return new TrelloImpl(appKey, token, a);
    }
}
